cfgmgr/natmgr.cpp

```diff
@@ -5049,7 +5049,7 @@ void NatMgr::doStaticNatTask(Consumer &consumer)
         /* Ensure the key size is 1 otherwise ignore */
         if (keys.size() != STATIC_NAT_KEY_SIZE)
         {
-            SWSS_LOG_ERROR("Invalid key size %lu, skipping %s", keys.size(), key.c_str());
+            SWSS_LOG_ERROR("Invalid key size %" PRIu32 ", skipping %s", keys.size(), key.c_str());
```

Comment:
Can you confirm it doesn't break for 64bit cpu. Also check %z as it can be used for size_t/size_type

Reply:
changed to %zu